data/README.md

```diff
@@ -0,0 +1,61 @@
+# The `data` Directory
+
+## Overview
+
+This directory contains files used for pre-loading our production server
+database with *Pseudomonas aeruginosa*-related samples and annotations. All of
+these files are loaded into the system using the management commands listed in
+[the load_default_pseudomonas_data.sh script](https://github.com/greenelab/adage-server/blob/master/load_default_pseudomonas_data.sh).
+These commands take the location and filename of the desired data files as
+an argument.
+
+If using [fabric](http://www.fabfile.org/), these can also be loaded using the
+`import_data_and_index()` function in
+[fabfile/adage_server.py](https://github.com/greenelab/adage-server/blob/master/fabfile/adage_server.py).
+The
+[adage/adage/configy.py.template file](https://github.com/greenelab/adage-server/blob/master/adage/adage/config.py.template)
+specifies which files are loaded by this `import_data_and_index()` function.
+
+All of the files below are tab-delimited text files. If you wish to load your
```

Comment:
The first sentence is not true any more because the gene-gene network data file is a gzipped file.

Reply:
It's true that it is a gzipped file because of the size, but the decompressed file we care about (and probably our users will too) is still a tab-delimited text file. If they want to use another file, it will still need to be tab-delimited. You bring up a good point about being more precise though, so I will put in a note to do that.